scripts/extract-hvd-content.ts

```diff
@@ -3,19 +3,24 @@ import path from 'path'
 import { fetchGithubArchiveZip } from 'lib/fetch-github-archive-zip'
 import { isDeployPreview } from 'lib/env-checks'
 
+export const BASE_REPO_CONFIG = {
+	owner: 'hashicorp',
+	ref: process.env.CURRENT_GIT_BRANCH || 'main',
+	repo: 'hvd-docs',
+	contentPath: '/content',
+}
+
 /**
  * Set the location in the local filesystem where we'll extract HVD content.
  */
-export const HVD_CONTENT_DIR = path.join(
+const HVD_REPO_DIR = path.join(
 	process.cwd(),
 	'src/.extracted/hashicorp-validated-designs'
 )
 
-const BASE_REPO_CONFIG = {
-	owner: 'hashicorp',
-	ref: 'main',
-	repo: 'hvd-docs',
-}
+export const HVD_CONTENT_DIR =
+	path.join(process.cwd(), process.env.LOCAL_CONTENT_DIR) ||
```

Comment:
This env is set in the `hvd-docs` build script to point to the local repo data (i.e. `../content`)